stdlib/src/builtin/float_literal.mojo

```diff
@@ -239,7 +239,38 @@ struct FloatLiteral(
             return truncated
         return truncated + 1
 
-    # TODO: implement __round__
+    @always_inline("nodebug")
+    fn __round__(self, ndigits: IntLiteral = 0) -> Self:
+        """Return the rounded value of the FloatLiteral.
+
+        Args:
+            ndigits: The number of digits to round to. Defaults to 0.
+        Returns:
+            The rounded value.
+        """
+        # Handle special values first.
+        if not self._is_normal():
+            return self
+
+        alias one = __mlir_attr.`#kgen.int_literal<1> : !kgen.int_literal`
+        alias ten = __mlir_attr.`#kgen.int_literal<10> : !kgen.int_literal`
+        var multiplier = one
+        for _ in range(ndigits):
+            multiplier = __mlir_op.`kgen.int_literal.binop`[
```

Comment:
Seems like this could be done with `IntLiteral.__pow__`. Could you please add a TODO: to change this once that's implemented?

Reply:
Done: https://github.com/msaelices/mojo/commit/c8bb29a69741a3e8c39f6be212cb0c29657dae10